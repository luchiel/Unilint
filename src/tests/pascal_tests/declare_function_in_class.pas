type
  a = class
    constructor Create;
    function GoodFunction: boolean;
  end;

function Luck: boolean;
begin
  return true;
end;

function a.GoodFunction: boolean;
  begin
    return true;
end;

begin
end.