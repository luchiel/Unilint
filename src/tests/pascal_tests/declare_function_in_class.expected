13:1: indent too deep
14:1: indent too deep
